app/src/main/java/com/alexbar/layoutbasic/movies_app/nav/BottomNavigationGraph.kt

```diff
@@ -4,20 +4,38 @@ import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
+import androidx.navigation.navArgument
+import androidx.navigation.navDeepLink
 import com.alexbar.layoutbasic.movies_app.model.Media
 import com.alexbar.layoutbasic.movies_app.screen.detail.MediaDetailScreen
 import com.alexbar.layoutbasic.movies_app.screen.favorites.FavoritesMediaScreen
+import com.alexbar.layoutbasic.movies_app.screen.login.LoginScreen
 import com.alexbar.layoutbasic.movies_app.screen.mix.MixMediaScreen
 import com.alexbar.layoutbasic.movies_app.screen.trending.TrendingMediaScreen
+import com.alexbar.layoutbasic.movies_app.utils.getImageBackdropCompleteUrl
+import com.alexbar.layoutbasic.movies_app.utils.getImagePosterCompleteUrl
 import com.alexbar.layoutbasic.movies_app.utils.getVoteAverageFormatted
 import com.alexbar.layoutbasic.ui.theme.AppBackground
+import java.net.URLEncoder
+import java.nio.charset.StandardCharsets
+
```

Comment:
```
object Screens {
    const val DetailMediaScreen = "detailMediaScreen"
    const val DetailMediaDeepLinkScreen = "detailMediaDeepLinkScreen"
    // Define other screens and their routes

    // Routes with arguments
    val DetailMediaScreen.routeWithArgs: String
        get() = "$DetailMediaScreen/{title}/{description}/{voteAverage}/{imageBackdropUrl}/{imagePosterUrl}"
}
```

this one as well is just for readability of the code , these are used for screen names and their routes to avoid hardcoding